sunpy/timeseries/sources/eve.py

```diff
@@ -75,21 +75,26 @@ def plot(self, axes=None, **kwargs):
         """
         self._validate_data_for_plotting()
         names = ("Flux \n 0.1-7nm", "Flux \n 18nm", "Flux \n 26nm", "Flux \n 30nm", "Flux \n 36nm")
-        colors = ('tab:blue', 'tab:orange', 'tab:green', 'tab:red', 'tab:purple', 'tab:brown', 'tab:pink', 'tab:gray', 'tab:olive', 'tab:cyan')
+        colors = ('tab:blue', 'tab:orange', 'tab:green', 'tab:red', 'tab:purple',
+                  'tab:brown', 'tab:pink', 'tab:gray', 'tab:olive', 'tab:cyan')
         predefined_axes = False
         if isinstance(axes, np.ndarray):
             predefined_axes = True
         elif axes is None:
             axes = self.to_dataframe().plot(subplots=True, sharex=True, **kwargs)
         for i, ax in enumerate(axes):
             if predefined_axes:
-                ax.plot(self._data[self._data.columns[i]], color=colors[i%len(colors)], label=self._data.columns[i])
+                ax.plot(self._data[self._data.columns[i]], color=colors[i %
+                        len(colors)], label=self._data.columns[i])
                 plt.xticks(rotation=30)
             ax.set_ylabel(names[i])
             ax.legend(loc="upper right")
         axes[-1].set_xlim(self.to_dataframe().index[0], self.to_dataframe().index[-1])
-        axes[-1].set_xlabel("Time (UT) " + str(self.to_dataframe().index[0])[0:11])
-        axes[-1].xaxis.set_major_formatter(dates.DateFormatter("%H:%M"))
+        # TODO: Work out a way to set this based on the timespan of the data.
```

Comment:
Maybe this is a dumb question, but why aren't we using `astropy.visualization.time_support`? Is this incompatible with pandas?

Reply:
Not a clue, never used it.

Reply:
We don't need it at all it seems. I have removed the whole dates conversion.